atc/api/resourceserver/clear_cache.go

```diff
@@ -0,0 +1,37 @@
+package resourceserver
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/concourse/concourse/atc/db"
+)
+
+func (s *Server) ClearResourceCaches(pipeline db.Pipeline) http.Handler {
+
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		resourceName := r.FormValue(":resource_name")
+
+		clearedVersions := []string{resourceName, "v1"}
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+
+		json.NewEncoder(w).Encode(clearedVersions)
```

Comment:
Returning the # of caches cleared would be nice; here's how the task cache clearing endpoint returns it:

https://github.com/concourse/concourse/blob/0e1b195bf964cf11e88c6cbd75775f5160cce1df/atc/api/jobserver/clear_task_cache.go#L58